### ISSUE TITLE
Job Points Modifications Reset at Server Restart

### ISSUE THREAD

**Issue by LCookman** [Zrips/Jobs#314]
So as the title states there is an issue with jobs points that when you add or subtract them they don't  persist after a restart. Specifically speaking about `/jobs points add` and the use of the Job Shop, which is how I first learned of this issue. 

**Steps to Recreate and Testing:**
So first I gave myself 10,000 jobs points with the command `/jobs points add creedthefreak 10000`. Whose effect can be seen from this image here. 

![beforerestart](https://user-images.githubusercontent.com/17609860/49917583-a2647280-fe54-11e8-9424-cf60af679c3c.png)

After I ran the command I disconnected and reconnected and yet the job points that were given were persisting after the disconnect. After the disconnect I restarted the server after running `save-all`. Once I logged back in after the server restarted the points had reset to what they were before I ran the `/jobs points add` command. Which can be seen below.

![afterrestart](https://user-images.githubusercontent.com/17609860/49920832-7bac3900-fe60-11e8-9de9-67f8125578a6.png)

This same type of issue occurred when using the Job Shop. Here is an image before a restart to the server when the user buys an item from the shop for 5000 points.

![shopafterpurchasenoreset](https://user-images.githubusercontent.com/17609860/49920897-b44c1280-fe60-11e8-8ca6-1b07e755f4c3.png)

So the points were taken from his player, and this change persisted still after a disconnect. When I restarted the server the points taken from the player were returned giving them the item and the job shop item they bought. This is the main issue which the player essentially gets a free item.

![shopafterpurchaseafterreset](https://user-images.githubusercontent.com/17609860/49920972-dcd40c80-fe60-11e8-80e3-4fdc66d83ba0.png)

We have also tried to update to the latest paper spigot version, which didn't fix the issue. We then tried to go back to vanilla spigot yet this bug still persisted. We believe the issue lies here in Jobs itself, if you would like more information on our setup or have more questions as to what is happening let me know, I am more than willing to help out and get this issue fixed.

So I didn't really notice any exceptions being thrown by Jobs at least not in the regular log files. Here they are just in case they are needed. One is before a restart and one is after a restart to the server.

**Comment by montlikadani** [comment]
This error has not occurred to me since I've just been testing it. I did not change anything in the code when I tested it. I tested this in 1.13 (Paper).
Take the following steps:
- Make sure you are using the latest (or DEV.) version of JobsReborn.
- If enabled, try to disable DBCleaning in the generalConfig file.
- Make sure that you use `Limit.Point` in the generalConfig file, if yes, try to disable it.

Such a command is **not** available at Jobs: `/jobs points add [playername] [amount]`
In the plugin, this command exists: `/jobs editpoints set/add/take [playername] [amount]`
